docs/Reference/Server.md

```diff
@@ -1024,13 +1024,20 @@ const defaultRoute = fastify.getDefaultRoute()
 <a id="setDefaultRoute"></a>
 
 **Note**: The default 404 handler, or one set using `setNotFoundHandler`, will
```

Comment:
I would be even more specific:
let's deprecate this and remove in the next major.z